less/gameboard.less

```diff
@@ -4,7 +4,7 @@
   bottom: 0;
   left: 0;
   right: 0;
-  top: 50px;
+  top: 40px;
```

Comment:
This has shown up a few times, is it the height of the nav bar? May be worth pulling into a variable (e.g. `@nav-bar-height`)